test/ui-test/src/screens/StatusWalletScreen.py

```diff
@@ -0,0 +1,123 @@
+    SEED_PHRASE_INPUT_3: str = "mainWindow_Add_Account_Popup_Seed_Phrase_3"
+    SEED_PHRASE_INPUT_4: str = "mainWindow_Add_Account_Popup_Seed_Phrase_4"
+    SEED_PHRASE_INPUT_5: str = "mainWindow_Add_Account_Popup_Seed_Phrase_5"
+    SEED_PHRASE_INPUT_6: str = "mainWindow_Add_Account_Popup_Seed_Phrase_6"
+    SEED_PHRASE_INPUT_7: str = "mainWindow_Add_Account_Popup_Seed_Phrase_7"
+    SEED_PHRASE_INPUT_8: str = "mainWindow_Add_Account_Popup_Seed_Phrase_8"
+    SEED_PHRASE_INPUT_9: str = "mainWindow_Add_Account_Popup_Seed_Phrase_9"
+    SEED_PHRASE_INPUT_10: str = "mainWindow_Add_Account_Popup_Seed_Phrase_10"
+    SEED_PHRASE_INPUT_11: str = "mainWindow_Add_Account_Popup_Seed_Phrase_11"
+
+
+class StatusWalletScreen:
+    
+    def acceptSigningPhrase(self):
+        click_obj_by_name(SigningPhrasePopUp.OK_GOT_IT_BUTTON.value)
+        
+    def addWatchOnlyAccount(self, account_name: str, address: str):
+        click_obj_by_name(MainWalletScreen.ADD_ACCOUNT_BUTTON.value)
+        
+        type(AddAccountPopup.ACCOUNT_NAME_INPUT.value, account_name)
+
+        click_obj_by_name(AddAccountPopup.ADVANCE_SECTION.value)
+        click_obj_by_name(AddAccountPopup.TYPE_SELECTOR.value)
+        time.sleep(1)
+        click_obj_by_name(AddAccountPopup.TYPE_WATCH_ONLY.value)
+        
+        type(AddAccountPopup.ADDRESS_INPUT.value, address)
+        click_obj_by_name(AddAccountPopup.ADD_ACCOUNT_BUTTON.value)
+
+    def importPrivateKey(self, account_name: str, password: str, private_key: str):
+        click_obj_by_name(MainWalletScreen.ADD_ACCOUNT_BUTTON.value)
+        
+        type(AddAccountPopup.PASSWORD_INPUT.value, password)
+        type(AddAccountPopup.ACCOUNT_NAME_INPUT.value, account_name)
+
+        click_obj_by_name(AddAccountPopup.ADVANCE_SECTION.value)
+        click_obj_by_name(AddAccountPopup.TYPE_SELECTOR.value)
+        time.sleep(1)
+        click_obj_by_name(AddAccountPopup.TYPE_PRIVATE_KEY.value)
+        
+        type(AddAccountPopup.PRIVATE_KEY_INPUT.value, private_key)
+        click_obj_by_name(AddAccountPopup.ADD_ACCOUNT_BUTTON.value)
+    
+    def importSeedPhrase(self, account_name: str, password: str, mnemonic: str):
+        click_obj_by_name(MainWalletScreen.ADD_ACCOUNT_BUTTON.value)
+        
+        type(AddAccountPopup.PASSWORD_INPUT.value, password)
+        type(AddAccountPopup.ACCOUNT_NAME_INPUT.value, account_name)
+
+        click_obj_by_name(AddAccountPopup.ADVANCE_SECTION.value)
+        click_obj_by_name(AddAccountPopup.TYPE_SELECTOR.value)
+        time.sleep(1)
+        click_obj_by_name(AddAccountPopup.TYPE_SEED_PHRASE.value)
+        
+        words = mnemonic.split()
+        scroll_obj_by_name(AddAccountPopup.SCROLL_BAR.value)
+        time.sleep(2)
+
+        scroll_obj_by_name(AddAccountPopup.SCROLL_BAR.value)
+        time.sleep(2)
+
+        scroll_obj_by_name(AddAccountPopup.SCROLL_BAR.value)
+        time.sleep(2)
+
+        type(AddAccountPopup.SEED_PHRASE_INPUT_0.value, words[0])
```

Comment:
**Suggestion:** Maybe here we could get the `grid` object that contains the `inputs` and do a loop getting its `itemAtIndex` instead of having each `input` separately in the `names.py` file and all these statements will be reduced to a `for` and one `type` call.

If you want, we can create a new task to address it separately.

Reply:
agree it is verbose but I think it is ok for test to be more verbose than code, as it really reflect what is happening